storage/README.md

```diff
@@ -28,6 +29,41 @@ amount of data at any time.
 
 ## Samples
 
+### ACL (Access Control Lists)
+
+View the [documentation][acl_docs] or the [source code][acl_code].
+
+__Usage:__ `node acl --help`
+
+```
+Commands:
```

Comment:
You mean auto-generate `README.md`? The usage information was generated, I just ran `node acl --help` and pasted the output into README.md. I used to have to write all that by hand, now most of README.md _is_ generated.